src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/UnusedImportsCheckTest.java

```diff
@@ -108,21 +108,21 @@ public void testProcessJavadoc() throws Exception {
     public void testProcessJavadocWithLinkTag() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(UnusedImportsCheck.class);
         final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
-        verify(checkConfig, getPath("InputUnusedImportWithValueTag.java"), expected);
+        verify(checkConfig, getPath("InputUnusedImportsWithValueTag.java"), expected);
     }
 
     @Test
     public void testAnnotations() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(UnusedImportsCheck.class);
         final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
-        verify(checkConfig, getNonCompilablePath("package-info.java"), expected);
+        verify(checkConfig, getNonCompilablePath("InputUnusedImportsAnnotations.java"), expected);
```

Comment:
ok, lets keep new name. Checkstyle do not care, it was just signal to used that file was weird because it is non java file actually.
`package-info.java` - is parse-able by us, that is why we process it. In other words , it does not make problems for us so we work with it.
`module-info.java` - is in the same family of file in java project. We do not process it, because it completely different by structure and .... .

so, to make it clear for future contributor, lets change comment content from `Annotated package definition to check if import is declared unused` --> `this file content is based on  package-info.java file structure`